src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -297,6 +436,18 @@ const PersonalInfoList = ({
       </Typography>
     ) : null;
 
+    const disclaimer2 =
+    !myProf &&
+    (isHomePhonePrivate ||
+      isAddressPrivate ||
+      isMobilePhonePrivate ||
+      isCampusLocationPrivate ||
+      isSpousePrivate) ? (
+      <Typography align="left" className="disclaimer">
+        Private by request, visible only to gordon police
```

Comment:
I think all of emergency contact information should be kept private by default, not by request.

Reply:
Thank you, that's a good idea!